src/main/java/mate/academy/internetshop/model/Item.java

```diff
@@ -41,4 +41,9 @@ public String toString() {
                 + name + "', price="
                 + price + "}";
     }
+
+    @Override
+    public int compareTo(Item item) {
```

Comment:
do we need this method?

Reply:
Item model class should implement Comparable interface and have compareTo() method Overriden so I can sort Item list by alphabetical order in getAllItems() Method